recipes/tests.py

```diff
@@ -33,22 +36,27 @@ def list_ingredient(self):
         return response
 
     def test_create_ingredient_with_user_authentication(self):
-        response = self.create_ingredient(self.test_ingredient_name)
+        user, client = self.create_user()
+        response = self.create_ingredient(self.test_ingredient_name, client)
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(response.data.get('name'), self.test_ingredient_name)
         self.assertIsNotNone(response.data.get('image'))
 
     def test_create_ingredient_without_user_authentication(self):
-        response = self.create_ingredient(self.test_ingredient_name, fail_authenticate=True)
+        user, client = self.create_user(fail_authenticate=True)
+        response = self.create_ingredient(self.test_ingredient_name, client)
```

Comment:
You do not require to create a user, you can send request without `HTTP_AUTHORIZATION` header.
Lets use this client `client = APIClient()`.